addons/web/static/tests/fields/field_utils_tests.js

```diff
@@ -297,7 +298,77 @@ QUnit.test('parse percentage', function(assert) {
 
     core._t.database.parameters = originalParameters;
 
-})
+});
+
+QUnit.test('parse datetime', function(assert) {
+    assert.expect(6);
+
+    var originalParameters = _.clone(core._t.database.parameters);
+    var done = assert.async();
+
+    var odooLocales = {
+        'en': {
+            date_format: '%m/%d/%Y',
+            time_format: '%H:%M:%S'
+        },
+        'fr': {
+            date_format: '%d/%m/%Y',
+            time_format: '%H:%M:%S'
+        },
+        'nb': {
+            date_format: '%d. %b %Y',
+            time_format: '%H:%M:%S'
+        }
+    }
+
+    var momentLocales = _.map(['en-gb', 'fr', 'nb'], function(lang) {
+        return ajax.loadJS('/web/webclient/locale/' + lang.replace('-', '_'));
```

Comment:
Not sure we can do that in a test.

Reply:
It is required otherwise the test is useless

Reply:
Yeah, but that means a RPC in a JS test. AFAIK, we don't do that, we mock instead.

Reply:
:+1: for the mock